packages/trader/src/Modules/Account/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -585,8 +586,12 @@ class PersonalDetailsForm extends React.Component {
                                 </FormBody>
                                 <FormFooter>
                                     {status && status.msg && <FormSubmitErrorMessage message={status.msg} />}
+                                    <div className='account-form__footer-note'>
+                                        Please make sure your information is correct or
+                                        it may affect your trading experience.
+                                    </div>
                                     <Button
-                                        className='account-form__footer-btn btn--primary--default'
+                                        className='account-form__footer-btn btn--primary--green'
```

Comment:
is the initial button color supposed to be green? or should it only be green when the form submission is successful?

Reply:
Yes ... you are right. thanks for your review